packages/preview/nth/0.2.0/nth.typ

```diff
@@ -0,0 +1,19 @@
+#let nth(ordinal-num) = {
+  let ordinal-str = str(ordinal-num)
+  let ordinal-suffix = if ordinal-str.ends-with(regex("1[0-9]")) {
+    "th"
+  }
+  else if ordinal-str.last() == "1" {
+    "st"
+  }
+  else if ordinal-str.last() == "2" {
+    "nd"
+  }
+  else if ordinal-str.last() == "3" {
+    "rd"
+  }
+  else {
+    "th"
+  }
+  show: ordinal-str + super(ordinal-suffix)
```

Comment:
This `show: ` here doesn't really do anything. It replaces the remainder of the block (nothing) with the content after the colon.

Reply:
You're right, I'll remove it.